source/d3d11/d3d11_device_context.cpp

```diff
@@ -847,10 +847,26 @@ void    STDMETHODCALLTYPE D3D11DeviceContext::ExecuteCommandList(ID3D11CommandLi
 #if RESHADE_ADDON
 	// Use secondary command list event here, since this may be called on deferred contexts as well
 	reshade::invoke_addon_event<reshade::addon_event::execute_secondary_command_list>(this, command_list_proxy);
+
+	// Emit RENDERSUBMIT_START/END for immediate context execution path. For deferred contexts this represents submission to immediate.
+	const bool is_immediate = (this == _device->_immediate_context);
+	if (is_immediate)
+	{
+		const uint64_t ts_begin = std::chrono::duration_cast<std::chrono::nanoseconds>(std::chrono::high_resolution_clock::now().time_since_epoch()).count();
+		reshade::invoke_addon_event<reshade::addon_event::latency_rendersubmit_start>(this, nullptr, ts_begin);
+	}
```

Comment:
Command list execution is not a very common occurence in D3D11 titles, and in those that make use of it, will be called multiple times per frame with various command lists. As in D3D12, this is not the right place to issue the `RENDERSUBMIT_START/END` marker every time.